lib/script/script.js

```diff
@@ -583,7 +591,7 @@ Script.buildMultisigOut = function(publicKeys, threshold, opts) {
  *
  * @param {PublicKey[]} pubkeys list of all public keys controlling the output
  * @param {number} threshold amount of required signatures to spend the output
- * @param {Array} signatures signatures to append to the script
+ * @param {Buffer[]} signatures and array of signature buffers to append to the script
```

Comment:
"Array" I think is accurate here, jsdoc won't parse this correctly with our documentation ([also see](https://github.com/bitpay/bitcore-mnemonic/commit/e530ff8b4d118fbbf033934f10466d804707f392#diff-da7cf14a5d0febb8b6aa96c9e244ce54R32))